Kitodo-ImageManagement/src/main/java/org/kitodo/imagemanagement/ConvertRunner.java

```diff
@@ -51,7 +51,7 @@ class ConvertRunner {
     /**
      * Default timeout.
```

Comment:
Even if we have a speaking variable name, "Default timeout (seconds)" or a similar description might help additionally.